scripts/dataset_creation/soda_audio.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Steps to reproduce the fixie-ai/soda-audio dataset
+
+# Step 0: create the `fixie-ai/soda-audio` dataset in the UI or using huggingface_hub.create_repo
+
+# Step 1: Create an alternative last turn using Llama3-8b Instruct model
+# We want the model to generate the same response whether the input is audio or text
+
+just ds_tool textgen -d allenai/soda --shuffle True -s test -n 1000 -u fixie-ai/soda-audio -c alt_last_turn \
+    -T @ultravox/tools/ds_templates/soda_alt_last_turn.jinja -j -b https://api.fireworks.ai/inference/v1 \
+    -k $FIREWORKS_API_KEY -m accounts/fireworks/models/llama-v3-8b-instruct --token $HF_WRITE_TOKEN
+
+just ds_tool textgen -d allenai/soda --shuffle True -s validation -n 1000 -u fixie-ai/soda-audio -c alt_last_turn \
+    -T @ultravox/tools/ds_templates/soda_alt_last_turn.jinja -j -b https://api.fireworks.ai/inference/v1 \
+    -k $FIREWORKS_API_KEY -m accounts/fireworks/models/llama-v3-8b-instruct --token $HF_WRITE_TOKEN
+
+just ds_tool textgen -d allenai/soda --shuffle True -s train -n 100000 -u fixie-ai/soda-audio -c alt_last_turn \
+    -T @ultravox/tools/ds_templates/soda_alt_last_turn.jinja -j -b https://api.fireworks.ai/inference/v1 \
+    -k $FIREWORKS_API_KEY -m accounts/fireworks/models/llama-v3-8b-instruct --token $HF_WRITE_TOKEN
+
+
+# Step 2: TTS the turn before last: audio input for the model
+
+just ds_tool tts -d fixie-ai/soda-audio -u fixie-ai/soda-audio -c "dialogue[-2]" -a audio_one_but_last -i eleven -V random --token $HF_WRITE_TOKEN -N 8
```

Comment:
So this adds `alt_last_turn` as an assistant completion, and also `audio_one_but_last` as a TTS for the last user message. `audio_one_but_last` is a bit inscrutable to me, maybe `audio_last_user_message`? (and maybe `alt_last_assistant_message`?)

Reply:
I also don't like the name. The issue is there is no notion of user or assistant here, so those names would just be confusing.

Reply:
hmm, agreed. `alt_last_turn` and `audio_second_last_turn` perhaps?

Reply:
Okay I'll change it. I'll have to reupload the dataset, so I'll block the PRs for that to land first.